### PR TITLE
Add showcase card for Suppliers list

### PR DESCRIPTION
**Pull request by sarjon**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Displays showcase card when there are not suppliers in shop.
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Partially https://github.com/PrestaShop/PrestaShop/issues/10545
| How to test?  | When there are no suppliers in shop, you should see nice showcase card about suppliers.

### Final result

suppliers

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/12498)

### CODE REVIEW

**Comment by sarjon** [comment]
@matks ready for review. :+1:

**Review by matks — DISMISSED** [review]
Perfect 🥇

**Comment by matks** [comment]
Except lint 🤣

**Comment by matks** [comment]
@TristanLDD This is the 1st Empty state being implemented (all other cards are Helper Cards, not Empty States) 🏅please have a look 😉

**Comment by TristanLDD** [comment]
Yeah ! :D 
Design validated, can't wait to see them all around the BO :)

File: src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Blocks/EmptyState/_default.html.twig [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<div class="text-center">
+  <p class="mb-0 mt-2"><i class="material-icons">warning</i></p>
+  <p class="mb-2">{{ 'No records found'|trans }}</p>
```
Comment (LouiseBonnard) [inline src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Blocks/EmptyState/_default.html.twig]: Please localize it in _Admin.Global_. It currently has no domains.

File: src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Blocks/EmptyState/supplier.html.twig [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<div class="text-center showcase-list-card">
+  <img class="img-responsive mt-3" src="{{ asset('themes/new-theme/img/empty_state/supplier.svg') }}">
+
+  <p class="mt-4 showcase-list-card__header">{{ 'Keep in touch with your suppliers'|trans({}, 'Admin.Catalog.Feature') }}</p>
+
+  <p class="mx-auto showcase-list-card__message">
+    {{ 'Having suppliers is optional if brands supply you directly. Make sure you don\'t confuse product suppliers and product brands to facilitate stock management.'|trans({}, 'Admin.Catalog.Feature') }}
```
Comment (LouiseBonnard) [inline src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Blocks/EmptyState/supplier.html.twig:32]: Is it okay for all that it is the shortened version of the wording?

Comment (LouiseBonnard) [reply src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Blocks/EmptyState/supplier.html.twig:32]: @TristanLDD

**Review by matks — DISMISSED** [review]

**Comment by marionf** [comment]
@sarjon 

Can we revert the image in RTL language ?

capture d ecran_1029

Also, I see a "don't show me again" link on the [invision mock-up](https://projects.invisionapp.com/share/JVN12YQSBU7#/screens/306924300) that isn't displayed currently:

capture d ecran_1034

Don't know if it should appear @TristanLDD  ?

**Comment by sarjon** [comment]
i cant see "don't show me again" link in zeplin. :confused:

**Comment by TristanLDD** [comment]
Hello @sarjon & @marionf, 

We decided to remove the link 'don't show me again' and to display the empty state every time the list is empty. The mockup on inVision was not updated as we work on Zeplin for this project, sorry for the confusion.

**Comment by marionf** [comment]
Thank you @TristanLDD 

@sarjon So, we have just the image to revert in RTL language

**Comment by sarjon** [comment]
@matks what's the correct place for adding RTL styles?  :thinking: i cannot find it. :/

**Comment by matks** [comment]
@sarjon `admin-dev/themes/new-theme/public/theme.rtlfix` 😉

**Comment by matks** [comment]
@sarjon It's strange, I fixed the same behavior in https://github.com/PrestaShop/PrestaShop/pull/12128 for the Customers helper card (mirror the image) but it seems I have not needed a dedicated RTL Fix.

Can you check ? I think it should be automatic.

**Comment by sarjon** [comment]
> Can you check ? I think it should be automatic.

ill double check that, because it wasnt working before, maybe i need to clear some cache.

**Comment by matks** [comment]
@sarjon When you set up a RTL language for BO, this will create a dedicated CSS file. So in order for new RTL CSS to be acknowledged, you need to remove this file and re-generate it.

You can do it by modifying your BO language "is it an RTL language" feature on and off.

**Comment by sarjon** [comment]
> You can do it by modifying your BO language "is it an RTL language" feature on and off.

did just that, removed all *_rtl.css from `themes/new-theme/public` and `themes/default/public` and toggled "Is RTL language" feature for my RTL language, css got regenerated and all BO looks good except for image. :/

**Comment by sarjon** [comment]
@matks this is how i fixed image. https://github.com/PrestaShop/PrestaShop/pull/12498/commits/57682b2c95427c94cd38aad4c7b6bed2e7bee0d4

@marionf image is fixed in RTL.

**Comment by marionf** [comment]
@sarjon I clean all caches but I don't see the picture reverted

capture d ecran_1089

And that's weird, because the menu should be on the right

On develop branch I well have the menu on the right

**Comment by marionf** [comment]
It's ok but I had to disable and re-enable the RTL button in the language

capture d ecran_1090

**Comment by sarjon** [comment]
> It's ok but I had to disable and re-enable the RTL button in the language

yeah, see @matks comments, it's kind of workaround to enable/disable language to clear RTL cache. :/

so, QA approved? :roll_eyes:

**Comment by marionf** [comment]
Yes :+1:

**Comment by matks** [comment]
Although I performed quite the same fix as for other helper cards behavior for RTL, it seems my code is gone :O
I created an issue to fix that https://github.com/PrestaShop/PrestaShop/issues/12715

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @sarjon !

**Pull request merged**